src/app/code-set/code-set-details/code-set-details.component.html

```diff
@@ -154,37 +152,30 @@
           </div>
           <h4
             class="inline-block marginless"
-            [ngClass]="{ deletedDataModelTitle: result?.deleted }"
+            [ngClass]="{ deletedDataModelTitle: codeSetDetail?.deleted }"
           >
             <span
               class="dataModelTypeIcon"
-              *ngIf="result?.domainType == 'CodeSet'"
+              *ngIf="codeSetDetail?.domainType == 'CodeSet'"
             >
               <i matTooltip="Code Set" class="fas fa-list"></i>
             </span>
             <mdm-inline-text-edit
-              [readOnly]="showEditDescription"
-              [inEditMode]="editableForm.visible"
-              [(ngModel)]="result.label"
+              [readOnly]="false"
+              [inEditMode]="editMode"
+              [showButtons]="true"
```

Comment:
Adding inline save/cancel buttons to the `<mdm-inline-text-edit>` is a good idea, makes the label editor in the model detail less bulky.